### ISSUE TITLE
Meta images data Downloading

### ISSUE THREAD

**Issue by NeverGiveU** [racinmat/mal-analysis#1]
Excuse me, is there any methods to download the meta anime images?

**Comment by racinmat** [comment]
Yes, the MAL's DNS has changed, but the relative url remained the same, so you can get the anime images by using the `image_url` column, but change the `myanimelist.cdn-dena.com` to `cdn.myanimelist.net` and it should work.
